bundles/io/org.eclipse.smarthome.io.voice/src/main/java/org/eclipse/smarthome/io/voice/internal/extensions/SayConsoleCommandExtension.java

```diff
@@ -91,14 +88,12 @@ protected void unsetItemRegistry(ItemRegistry itemRegistry) {
      * for a given platform.
      *
      * @param context the bundle context to access the OSGi service registry
-     * @param os a valid osgi.os string value or "any" if service should be platform-independent
      * @return a service instance or null, if none could be found
      */
-    static private TTSService getTTSService(BundleContext context, String os) {
+    static private TTSService getTTSService(BundleContext context) {
```

Comment:
Could we use the "common" modifiers order? (http://checkstyle.sourceforge.net/config_modifier.html)